src/charts/bar.js

```diff
@@ -977,7 +1002,7 @@ define(function(require) {
         /**
          * Changes the order of items given the custom function
          * @param  {Function} _x             A custom function that sets logic for ordering
-         * @return { (Function | Module) }   Updated module with ordering function applied
+         * @return {(Function | Module)}   Updated module with ordering function applied
```

Comment:
Same here. Returns either the ordering function or the module with ordering function applied

Reply:
Nice catch! Will fix those